CSA.md

```diff
@@ -1,215 +1,221 @@
+
+    7.4 <span class="header_3">Provider Warranty Remedy</span>.  If <span class="keyterms_link">Provider</span> breaches a warranty in Section 7.3, <span class="keyterms_link">Customer</span> must give <span class="keyterms_link">Provider</span> notice (with enough detail for <span class="keyterms_link">Provider</span> to understand or replicate the issue) within 45 days of discovering the issue. Within 45 days of receiving sufficient details of the warranty issue, <span class="keyterms_link">Provider</span> will attempt to restore the general functionality of the Cloud Service or reperform the <span class="orderform_link">Professional Services</span>. If <span class="keyterms_link">Provider</span> cannot resolve the issue, <span class="keyterms_link">Customer</span> may terminate the affected Order Form and <span class="keyterms_link">Provider</span> will pay to <span class="keyterms_link">Customer</span> a prorated refund of prepaid fees for the remainder of the <span class="orderform_link">Subscription Period</span>. <span class="keyterms_link">Provider's</span> restoration and reperformance obligations, and <span class="keyterms_link">Customer's</span> termination right, are <span class="keyterms_link">Customer's</span> only remedies if <span class="keyterms_link">Provider</span> does not meet the warranties in Section 7.3.
 
 8. ## Disclaimer of Warranties  
-<span class="coverpage_link">Provider</span> makes no guarantees that the Cloud Service will always be safe, secure, or error-free, or that it will function without disruptions, delays, or imperfections. The warranties in Section 7.2 do not apply to any misuse or unauthorized modification of the Cloud Service, nor to any product or service provided by anyone other than <span class="coverpage_link">Provider</span>. <span class="coverpage_link">Provider</span> and <span class="coverpage_link">Customer</span> each disclaim all warranties, whether express or implied, including the implied warranties of merchantability, fitness for a particular purpose, title, and non-infringement. These disclaimers apply to the maximum extent permitted by Applicable Laws.
+<span class="keyterms_link">Provider</span> makes no guarantees that the Product will always be safe, secure, or error-free, or that it will function without disruptions, delays, or imperfections. The warranties in Section 7.2 do not apply to any misuse or unauthorized modification of the Product, nor to any product or service provided by anyone other than <span class="keyterms_link">Provider</span>. Except for the warranties in Section 7, <span class="keyterms_link">Provider</span> and <span class="keyterms_link">Customer</span> each **disclaim all other warranties, whether express or implied, including the implied warranties of merchantability, fitness for a particular purpose, title, and non-infringement. These disclaimers apply to the maximum extent permitted by Applicable Laws.**
```

Comment:
@tblt 's 7.2 => 7.3 was missed 
```suggestion
<span class="keyterms_link">Provider</span> makes no guarantees that the Product will always be safe, secure, or error-free, or that it will function without disruptions, delays, or imperfections. The warranties in Section 7.3 do not apply to any misuse or unauthorized modification of the Product, nor to any product or service provided by anyone other than <span class="keyterms_link">Provider</span>. Except for the warranties in Section 7, <span class="keyterms_link">Provider</span> and <span class="keyterms_link">Customer</span> each **disclaim all other warranties, whether express or implied, including the implied warranties of merchantability, fitness for a particular purpose, title, and non-infringement. These disclaimers apply to the maximum extent permitted by Applicable Laws.**
```